webpack/stories/decorators/withCardsDecorator.js

```diff
@@ -4,7 +4,7 @@ export const withCardsDecorator = storyFn => (
   <div
     style={{
       width: '100%',
-      height: '100vh',
+      height: '100%',
```

Comment:
I encountered same problem in my backport. Not sure why spellcheck doesn't like this string, but I solved it by adding "vh" to list of ignored words.

Reply:
yeah, I was about to add it, although on a second thought it means the same in this case :)